packages/bloc_lint/pubspec.yaml

```diff
@@ -12,7 +12,7 @@ environment:
   sdk: ">=3.7.0 <4.0.0"
 
 dependencies:
-  _fe_analyzer_shared: ">=92.0.0 <93.0.0"
+  _fe_analyzer_shared: ">=91.0.0 <93.0.0"
```

Comment:
This isn’t necessary. You can use the previous version of bloc_lint if you’re using flutter stable. Once flutter stable supports the latest analyzer you’ll be able to upgrade to the latest version of bloc_lint